docs/Customs Declare Exports AutoComplete.js

```diff
@@ -439,6 +419,10 @@ function holderOfAuthorisation(){
                 break;
         }
     }
+    if (currentPageIs("/customs-declare-exports/declaration/authorisations-required")) {
+        document.getElementById('code_no').checked = 'checked';
+        document.getElementById('submit').click();
+    }
```

Comment:
Do we need case for the remove page?

Reply:
Do we do that for other pages that are not on the direct path from start -> finish?

Reply:
I don't know, I just thought about remove page, not sure why